Framework/Source/Sample.h

```diff
@@ -161,17 +161,60 @@ namespace Falcor
         float mCurrentTime = 0;                           ///< Global time
         ArgList mArgList;
 
+		//	The Memory Check for one frame.
+		struct MemoryCheck
+		{	
+			unsigned long pFrame;
```

Comment:
Please look at the coding convention document for how we use variables. We don't use the regular C types, we use int32_t, uint32_t, etc.